server/routes/room_routes.py

```diff
@@ -0,0 +1,17 @@
+from flask import Blueprint, Response, request
+from bson import json_util
+import sys
+sys.path.insert(0, './controller')
+from room_controller import RoomController
+room_bp = Blueprint('room_bp', __name__)
+
+@room_bp.route('/api/room',methods=['GET'])
+def get_room():
+    room_id = request.args.get('room_id')
+    room_controller=RoomController()
+    room_data=room_controller.get_room_details(room_id)
```

Comment:
here are we getting a particular room, or all the available room types?
Need to discuss on this